container-action/entrypoint.sh

```diff
@@ -5,16 +5,18 @@ RSKJ_BRANCH="${INPUT_RSKJ_BRANCH}"
 POWPEG_NODE_BRANCH="${INPUT_POWPEG_NODE_BRANCH}"
 RIT_BRANCH="${INPUT_RIT_BRANCH}"
 LOG_LEVEL="${INPUT_RIT_LOG_LEVEL}"
+REPO_OWNER="${INPUT_REPO_OWNER:-rsksmart}" # Default to 'rsksmart' if not provided
 
 echo -e "\n\n--------- Input parameters received ---------\n\n"
 echo "RSKJ_BRANCH=$RSKJ_BRANCH"
 echo "POWPEG_NODE_BRANCH=$POWPEG_NODE_BRANCH"
 echo "RIT_BRANCH=$RIT_BRANCH"
 echo "LOG_LEVEL=$LOG_LEVEL"
+echo "REPO_OWNER=$REPO_OWNER"
```

Comment:
@rmoreliovlabs , as external PRs can also be created against the PowPeg repo, we may also need to clone `powpeg-node` repo with `$REPO_OWNER`. And then fallback to `rsksmart`, if that fails, wdyt?

eg.:
```
git clone "https://github.com/$REPO_OWNER/rskj.git" rskj || git clone "https://github.com/rsksmart/rskj.git" rskj

git clone "https://github.com/$REPO_OWNER/powpeg-node.git" rskj || git clone "https://github.com/rsksmart/powpeg-node.git" rskj
```

Reply:
or even better, you could check if both: external repo + branch exists, eg.:

```
          IS_RSKJ_BRANCH=$(git ls-remote https://github.com/$REPO_OWNER/rskj.git "$RSKJ_BRANCH")
          if test -n "${IS_RSKJ_BRANCH}"; then
            echo "Found matching branch name in $REPO_OWNER/rskj.git repo"
            git clone "https://github.com/$REPO_OWNER/rskj.git" rskj
          else
            echo "Found matching branch name in rsksmart/rskj.git repo"
            git clone "https://github.com/rsksmart/rskj.git" rskj
          fi
```

Reply:
Addressed, implemented your suggestion for both RSKJ and POWPEG repos, thanks!

Reply:
@rmoreliovlabs, that might not work if expected branch is missing in the counterpart forked repo. Check [this](https://github.com/rsksmart/rootstock-integration-tests/pull/103#discussion_r1786088735) comment